bigbluebutton-html5/imports/api/users/server/methods/userLeaving.js

```
@@ -15,19 +16,22 @@ export default function userLeaving(credentials, userId, connectionId) {
   check(requesterUserId, String);
   check(userId, String);
 
+  const meeting = Meetings.findOne({ meetingId });
+  if (!meeting) return false;
+
   const selector = {
     meetingId,
     userId,
   };
 
-  const User = Users.findOne(selector);
+  const user = Users.findOne(selector);
 
-  if (!User) {
+  if (!user) {
     throw new Meteor.Error('user-not-found', `Could not find ${userId} in ${meetingId}: cannot complete userLeaving`);
```

Just change this to a skip log instead of silently returning.

---

`Logger.info('Skipping userLeaving. Could not find ${userId} in ${meetingId}')`